### DISCUSSION TITLE
Zoom In is limited?

### DISCUSSION THREAD

**Discussion by AmitLavi-LeanAI** [SkalskiP/make-sense#349] [Q&A] [answered]
Hi, I want to be able to zoom in more. is this possible?
Thanks

**Comment by macdatar** [comment]
Hello, I would like to do the same. Did you ever figure out how it could be done?

**Reply by SkalskiP** [reply]
Hi, @AmitLavi-LeanAI and @macdatar, do you use `makesense.ai` on the web or host it locally?

**Reply by macdatar** [reply]
Using it locally because my files are large and they werent loading online.

**Reply by SkalskiP** [reply, marked as answer]
In that case, update the value of `MAX_ZOOM` to a larger value locally in your code.

https://github.com/SkalskiP/make-sense/blob/ee5ee68b439703dc5820f18f614d773580f7c464/src/settings/ViewPointSettings.ts#L4

**Reply by macdatar** [reply]
Thank you so much!
